client/app/dbaas/logs/detail/roles/logs-roles.service.js

```diff
@@ -121,10 +125,10 @@ class LogsRolesService {
         });
     }
 
-    _handleSuccess (serviceName, operation, successMessage) {
+    _handleSuccess (serviceName, operation, successMessage, name) {
         this.poller = this._pollOperation(serviceName, operation);
         return this.poller.$promise
-            .then(this.ServiceHelper.successHandler(successMessage));
+            .then(() => this.CloudMessage.success(this.$translate.instant(successMessage, { name })));
```

Comment:
why can't you use this.ServiceHelper.successHandler?

this.ServiceHelper.successHandler(successHandler)(name) should work :)

Reply:
ServiceHelper successHandler was giving $translateSanitization issue. So i switched it. ill try again and see.

Reply:
Have fixed this. Merging it